R/pp_check.R

```diff
@@ -52,18 +49,20 @@
 #'
 #' @examples
 #' library(performance)
-#' model <- lm(Sepal.Length ~ Species * Petal.Width + Petal.Length, data = iris)
-#' if (require("ggplot2") && require("see")) {
-#'   pp_check(model)
+#' model <- lm(mpg ~ disp, data = mtcars)
+#' if (require("see")) {
+#'   check_predictions(model)
 #' }
 #' @export
-pp_check <- function(object, ...) {
-  UseMethod("pp_check")
+check_predictions <- function(object, iterations = 50, check_range = FALSE, re_formula = NULL, ...) {
+  if (isTRUE(insight::model_info(object, verbose = FALSE))) {
```

Comment:
I just realized this code line here - what are you expecting from `model_info()` to be returned? I think this if-statement currently is never true.

Reply:
@bwiernik I think this should be checking for Bayesian methods, right?